src/python/scripts/dx-generate-dxapp

```diff
@@ -330,6 +331,9 @@ def main():
     parser.add_argument(
         '--release', default="14.04",
         help="Distribution release to use for the machine image.")
+    parser.add_argument(
+        '--version', default="0",
+        help="Version of the application execution environment inside the runSpec block.")
```

Comment:
This name of the variable may conflict with "version" of the app, if we ever want to introduce it here. I'd change it into e.g. runspec-version.

Reply:
done